### PR TITLE
LPS-164043 Check if variable is null to prevent NullPointerException

### PR DESCRIPTION
**Pull request by marcogalluzzi**

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp [hunk 1]
```diff
@@ -151,7 +151,14 @@ SearchContainer<?> searchContainer = new SearchContainer(renderRequest, itemSele
 						).put(
 							"uploadItemReturnType", HtmlUtil.escapeAttribute(returnType)
 						).put(
-							"uploadItemURL", uploadURL.toString()
+							"uploadItemURL",
+							() -> {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp:155]: good point here

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp (hunk 1 above)
Comment (marcogalluzzi) [reply modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp:155]: OK

**Review by AliciaGarciaGarcia — APPROVED** [review]
LGTM

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**